instance/jmsml_D_Land_Unit.xml

```diff
@@ -2504,36 +2548,60 @@
     </LegacySymbol>
     <LegacySymbol ID="FIRES_AIR_DEFENSE_COMPOSITE_SYM" Label="S*GPUCDC--*****" Remarks="Retired">
       <LegacyFunctionCode Name="2525C">UCDC--</LegacyFunctionCode>
-      <LegacyEntity ID="FIRES_AIR_DEFENSE_COMPOSITE_LE" Label="WAR.GRDTRK.UNT.CBT.ADF.CMPS" CloverGraphic="S-G-UCDC--_0.svg" RectangleGraphic="S-G-UCDC--_1.svg" SquareGraphic="S-G-UCDC--_2.svg" DiamondGraphic="S-G-UCDC--_3.svg" Icon="FULL_FRAME">
+      <LegacyEntity ID="FIRES_AIR_DEFENSE_COMPOSITE_LE" Label="WAR.GRDTRK.UNT.CBT.ADF.CMPS" Icon="Full_Frame" CloverGraphic="S-G-UCDC--_0.svg" RectangleGraphic="S-G-UCDC--_1.svg" SquareGraphic="S-G-UCDC--_2.svg" DiamondGraphic="S-G-UCDC--_3.svg" Icon="FULL_FRAME">
```

Comment:
@Dbarnes1 - Icon=FULL_FRAME appears twice in this line.  It only needs to be there once with the full caps.  Doesn't really matter where in the line.